enterprise/server/workflow/service/service.go

```diff
@@ -416,6 +561,9 @@ func (ws *workflowService) createActionForWorkflow(ctx context.Context, wf *tabl
 		DoNotCache:      true,
 	}
 	actionDigest, err := cachetools.UploadProtoToCAS(ctx, cache, instanceName, action)
+	if err != nil {
+		return nil, err
+	}
 	return actionDigest, err
```

Comment:
I don't fully understand this change ^. Why not:

return cachetools.UploadProtoToCAS(ctx, cache, instanceName, action)

Reply:
Removed the extraneous `err != nil` check. But I like the existing `actionDigest, err := ...` followed by `return actionDigest, err` because it makes the code more self-documenting.